man/man8/zpoolconcepts.8

```diff
@@ -93,6 +93,64 @@ compromised.
 The minimum number of devices in a raidz group is one more than the number of
 parity disks.
 The recommended number is between 3 and 9 to help increase performance.
+.It Sy draid , draid1 , draid2 , draid3
+A variant of raidz which provides integrated distributed hot spares that allow
+faster resilvering while retaining the benefits of raidz.  A dRAID vdev is
+constructed from multiple internal raidz groups, each with D data devices
+and P parity devices which are distributed over the children.  This allows
+all of the children to participate and for fully sequential resilvering.
+.Pp
+This fixed stripe width significantly effects both usable capacity and IOPs.
+For example, with the default D=8 and 4k disk sectors the minimum allocation
+size is 32k.  If using compression this relatively large allocation size can
+reduce the effective compression ratio.  When using ZFS volumes and dRAID the
+default volblocksize property is increased to account for the allocation
+size. Use of a mirrored
+.Sy special
+vdev type to store small blocks is recommended when using dRAID.
+.Pp
+In regards to IO/s, performance is similar to raidz since for any read all
+D disks must be accessed.  Delivered random IOPs can be reasonably
+approximated as floor(D/<data disks>)*<single-drive-IOPs>.
```

Comment:
suggest using the same letters as below: `draid with N disks of size X, D data disks, P parity level, and S spares`.  So I think this would be `floor((N-S)/(D+P))*`.

But now I've confused myself about what `D` is.  If it is not the "The number of data devices per redundancy group", specified by e.g. `draid:8D`, then I'd suggest using a different letter.

Reply:
You're right I've updated the approximate IOPs values to use the letters.  D is the number of data devices per redundancy group, D, P, and S are the same values used in `draid<P>:#<D>:#<S>`.